### ISSUE TITLE
Associated items to entries/categories

### ISSUE THREAD

**Issue by luxlacis** [klikli-dev/modonomicon#259]
It would be helpful to have certain items associated with an entry or category similar to how Lavender does it (see image). This way for new items that are added to the game players can quickly navigate to a specific entry via an item that would give them information.

An entry called "my first page" which is linked to nether quartz ore.
![associated-entry-tooltip](https://github.com/user-attachments/assets/618c2d16-8985-4585-8450-bbde04be77b7)

**Comment by klikli-dev** [comment]
This is a neat idea, when I find the time I will add it. 

I need to come up with a performant way to do this check, but I suppose we could build a map in a tooltip handler.
The main thing is to figure out how to modify tooltips on fabric, on neo (and forge) there's an event, I assume for fabric too

**Comment by luxlacis** [comment]
> This is a neat idea, when I find the time I will add it.
> 
> I need to come up with a performant way to do this check, but I suppose we could build a map in a tooltip handler. The main thing is to figure out how to modify tooltips on fabric, on neo (and forge) there's an event, I assume for fabric too

Yeah, I think Patchouli does this too but in a different way. It also shows a tooltip in-world when holding the book and looking at a block that is "associated" with an entry.

**Comment by DaFuqs** [comment]
Fabric API also has an event that triggers for item tooltips ✅ 

Storing item>page associations in a static map sounds like a very easy and straightforward approach.
What to do if an item has two values, though?

**Comment by luxlacis** [comment]
Any news on this?

**Comment by klikli-dev** [comment]
I have not progressed on it yet as recently my time has gone into porting to newer MC versions, but I still want to add the feature!
